### PR TITLE
Assignment Table Styling

### PR DESCRIPTION
**Pull request by christinem**
This PR restyles the assignment and dashboard table on both the student and admin/ta side so peer review assignments are now grouped together with their original assignments in the table.

### CODE REVIEW

File: app/views/assignments/_assignment_summary.html.erb [hunk 1]
```diff
@@ -2,8 +2,8 @@
   <div class='assignment_summary'>
     <h2 class='title'>
       <%= link_to I18n.t(:assignment_link,
-                         short_id: assignment.short_identifier,
-                         description: assignment.description),
+                         short_id: assignment.is_peer_review? ? assignment.short_identifier[0,2] : assignment.short_identifier,
```
Comment (david-yz-liu) [inline app/views/assignments/_assignment_summary.html.erb]: Don't ever use the peer review short_identifier. Use the parent assignment.

**Comment by christinem** [comment]
@david-yz-liu Fixed the merge conflicts in this branch.

**Comment by david-yz-liu** [comment]
@christinem the peer review tab for students needs to be hidden when the PR assignment is hidden.

**Comment by david-yz-liu** [comment]
Also the peer review short identifier appears in other tabs still, including the Reviews table.

**Comment by christinem** [comment]
@david-yz-liu I fixed the peer review tab comment. As for the short identifier, I fixed most of those in #2704. The only one I didn't change yet is the short identifier on the edit assignment page, as I'm not sure whether that one should be changed.

**Pull request merged**